agent/stats.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 Drone.IO Inc.
```

Comment:
Shouldn't this file be called `state`?

Reply:
I copied the name from those commits. state makes more sense, so I'll move it momentarily.